src/frontend/src/optimizer/plan_node/stream_global_approx_percentile.rs

```diff
@@ -95,8 +97,42 @@ impl PlanTreeNodeUnary for StreamGlobalApproxPercentile {
 impl_plan_tree_node_for_unary! {StreamGlobalApproxPercentile}
 
 impl StreamNode for StreamGlobalApproxPercentile {
-    fn to_stream_prost_body(&self, _state: &mut BuildFragmentGraphState) -> PbNodeBody {
-        todo!()
+    fn to_stream_prost_body(&self, state: &mut BuildFragmentGraphState) -> PbNodeBody {
```

Comment:
It seems we don't have any shuffle between the global approx percentile and the local approx percentile?

Reply:
Didn't quite get it. We do have shuffle between local -> global approx percentile. May take a look at the output inside `agg.yaml`. There we have:

```
      ├─StreamGlobalApproxPercentile { quantile: 0.8:Float64, relative_error: 0.01:Float64 }
      │ └─StreamExchange { dist: Single }
      │   └─StreamLocalApproxPercentile { percentile_col: $expr1, quantile: 0.8:Float64, relative_error: 0.01:Float64 }
```

Reply:
Oh, I see it. My main branch is out of date.

Reply:
BTW, I think `StreamMerge` inputs could be changed from binary inputs to multi inputs like `StreamUnion`. Maybe in next PR.